enterprise/server/cmd/executor/executor.go

```diff
@@ -297,3 +292,29 @@ func main() {
 	}()
 	env.GetHealthChecker().WaitForGracefulShutdown()
 }
+
+func cleanBuildRoot() {
+	rootDir := runner.GetBuildRoot()
+	entries, err := os.ReadDir(rootDir)
+	if err != nil {
+		if os.IsNotExist(err) {
+			return
+		}
+		log.Warningf("Failed to read build root dir: %s", err)
+		return
+	}
+
+	for _, entry := range entries {
+		// TODO: move cached firecracker/ociruntime images to a different
```

Comment:
mind filing a bug and referencing it here?

Reply:
Done